.github/workflows/build-and-test.yaml

```diff
@@ -46,3 +46,17 @@ jobs:
           files: ${{ steps.test.outputs.coverage-report-files }}
           fail_ci_if_error: false
           verbose: true
+          flags: total
+
+      - name: Move directory
+        run: |
+          mv lcov lcov-total || mkdir lcov-total
+          mv coveragepy coveragepy-total || mkdir coveragepy-total
```

Comment:
I don't want to add this kind of job in `build-and-test.yaml` because it's a detail and should be hidden in some action.

Reply:
Yes. After we agree on a design, I will move it to a new action in `autoware-github-actions`.